client/tests/testgeometry/test_utils.py

```diff
@@ -28,3 +30,34 @@ def test_incircle(self):
         self.assertLess(incircle(a, b, c, Vector(-11, 0)), 0)
         self.assertEqual(incircle(a, b, c, Vector(1, 1)), 0)
         self.assertEqual(incircle(a, b, c, Vector(1, -1)), 0)
+
+    def test_aabb(self):
```

Comment:
This is not part of the util tests, don't put it here. Create a seperate class and module named `test_aabb.py`